addons/digest/data/digest_tips_data.xml

```diff
@@ -9,100 +9,66 @@
             <field name="kpi_res_users_connected">True</field>
             <field name="kpi_mail_message_total">True</field>
         </record>
-    </data>
 
-    <data>
         <record id="digest_tip_digest_0" model="digest.tip">
-            <field name="name">Invite Team Members</field>
-            <field name="sequence">1</field>
-            <field name="group_id" ref="base.group_erp_manager"/>
+            <field name="name">Tip: Speed up your workflow with shortcuts</field>
+            <field name="sequence">800</field>
+            <field name="group_id" ref="base.group_user" />
             <field name="tip_description" type="html">
-<div>
-    <b class="font-16" style="font-size: 20px;line-height: 30px">Invite Team Members</b>
-    <p class="font-12" style="color:#888888;font-size: 15px; line-height: 25px; margin-top:0;">Go to your general settings to invite new users to your database.</p>
-    <img src="/digest/static/src/img/tips-digest-invite-team.jpg" style="max-width: 580px" width="100%"/>
-</div>
+                <div>
```

Comment:
Global remark: de-indent html content as pointed out hereabove to gain space. To propagate everywhere.